multihost_test/Sanity/test_pam_pwquality.py

```diff
@@ -0,0 +1,149 @@
+
+import pytest
+import subprocess
+import os
+
+
+def exceute_cmd(multihost, command):
+    cmd = multihost.client[0].run_command(command)
+    return cmd
+
+
+@pytest.mark.tier2
+class TestPamBz(object):
```

Comment:
Are these tests for a bug? I did not see a bugzilla in docstrings. If not can the class name be like "TestPamPwquality"

Reply:
DOne